torch/csrc/jit/tracer.cpp

```diff
@@ -5,6 +5,12 @@
 #include "torch/csrc/autograd/function.h"
 #include "torch/csrc/autograd/python_engine.h"
 #include "torch/csrc/autograd/functions/special.h"
+#include "torch/csrc/utils/auto_gil.h"
+#include "torch/csrc/utils/python_strings.h"
+
+#include <Python.h>
```

Comment:
nit: Python.h needs to be at the top of the file for silly reasons (https://stackoverflow.com/questions/10056393/g-with-python-h-how-to-compile)

Reply:
Oh actually it's already there. I'll just get rid of this one. How about frameobject and patchlevel? Are those okay here?